Lib/difflib.py

```diff
@@ -1187,7 +1187,11 @@ def unified_diff(a, b, fromfile='', tofile='', fromfiledate='',
         first, last = group[0], group[-1]
         file1_range = _format_range_unified(first[1], last[2])
         file2_range = _format_range_unified(first[3], last[4])
-        yield '@@ -{} +{} @@{}'.format(file1_range, file2_range, lineterm)
+        if isinstance(a, list) or isinstance(a, tuple) and \
+            isinstance(b, list) or isinstance(b, tuple):
+            yield '{}'.format(lineterm)
+        else:
+            yield '@@ -{} +{} @@{}'.format(file1_range, file2_range, lineterm)
```

Comment:
I propose to use interpolated strings to improve readability.

```suggestion
            yield f'{lineterm}'
        else:
            yield f'@@ -{file1_range} +{file2_range} @@{lineterm}'
```

Reply:
Fair enough, but that's outside the scope of this BPO, because I'm not making any changes in difflib, and I'm not otherwise making changes to this part of code.